Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -52,12 +52,9 @@
   id: PerforatedMetallic
   coefficients:
     Blunt: 2
-    Slash: 0.6
     Piercing: 0.2
-    Shock: 1.2
+    Shock: 0.6
   flatReductions:
-    Blunt: 5
-    Heat: 5
     Piercing: 10
```

Comment:
Doesn't this mean humanoids can break it with fists.

Reply:
its a flimsy mesh, sure you couldnt punch it apart but you could easily grab it and lean back to take it apart

Reply:
I pinged emissive on DISCORD so she what she says.